sched/task/task_posixspawn.c

```diff
@@ -184,7 +190,7 @@ static int nxposix_spawn_proxy(int argc, FAR char *argv[])
       /* Start the task */
 
       ret = nxposix_spawn_exec(g_spawn_parms.pid, g_spawn_parms.u.posix.path,
-                               g_spawn_parms.attr, g_spawn_parms.argv);
+                               g_spawn_parms.attr, g_spawn_parms.argv, NULL);
```

Comment:
Here is another place I'm not sure how to fix. The implementation seems quite messy, using a global parameter structure to pass parameters etc.